definitions/npm/@testing-library/react_v10.x.x/flow_v0.104.x-/test_react_v10.x.x.js

```diff
@@ -104,8 +105,6 @@ describe('waitForElement', () => {
 
 describe('waitForElementToBeRemoved', () => {
   it('should fail on invalid inputs', () => {
-    // $ExpectError
-    waitForElementToBeRemoved(1);
```

Comment:
That's the problem, it's now too loose and let you use anything.

Reply:
I don't disagree, what's an alternative way to type this?

Reply:
Like it was before, two separate declarations.